packages/programs/mud.config.ts

```diff
@@ -0,0 +1,54 @@
+import { defineWorld } from "@latticexyz/world";
+
+export default defineWorld({
+  // Note: this is required as the world is deployed with this
+  deploy: {
+    upgradeableWorldImplementation: true,
+  },
+  namespace: "default-1",
```

Comment:
should we come up with some convention for versioning these?

Reply:
ye can use a convention, used `-1` as I saw it in `mock-apps`